packages/fxa-auth-server/config/index.js

```diff
@@ -717,6 +717,7 @@ const conf = convict({
       doc: 'A shared secret to authenticate Stripe webhook requests',
     },
     stripeCustomerCacheTtlSeconds: {
+      // Deprecated
```

Comment:
Sounds good. I wasn't sure if it was okay to remove it 😅

Reply:
I think it's okay to remove, but also with following up by adding a note to the deploy doc to remove the config on the server for the train where this will ship out